act-rules-format.bs

```diff
@@ -699,6 +714,128 @@ This section is *non-normative*.
 }
 ```
 
+**Example 5:** Assumed 'Context' for this section
+
+```javascript
+{
+  "@vocab": "http://www.w3.org/ns/earl#",
+
+  "cnt": "http://www.w3.org/2011/content#",
+  "dct": "http://purl.org/dc/terms/",
+  "earl": "http://www.w3.org/ns/earl#",
+  "foaf": "http://xmlns.com/foaf/spec/",
+  "htp": "http://www.w3.org/2011/http#",
+  "ptr": "https://www.w3.org/2009/pointers#",
+  "schema": "http://schema.org/",
+  "xsd": "https://www.w3.org/2001/XMLSchema#",
+
+  "WCAG20": "https://www.w3.org/TR/WCAG20#",
+  "WCAG21": "https://www.w3.org/TR/WCAG21#",
+  "ACT-CG": "https://act-rules.github.io/",
+
+  "WebSite": "schema:WebSite",
+  "WebPage": "schema:WebPage",
+
+  "title": "dct:title",
+  "description": "dct:description",
+  "date": "dct:date",
+  "hasVersion": "dct:hasVesion", 
+  "isPartOf": "dct:isPartOf", 
+  "hasPart": "dct:hasPart",
+  "source": "dct:source",
+
+  "Agent": "foaf:term_Agent",
+  "Person": "foaf:term_Person",
+  "Organization": "foaf:term_Organization",
+  "Group": "foaf:term_Group",
+  "Document": "foaf:term_Document",
+  "name": "foaf:term_name",
+  "firstName": "foaf:term_firstName",
+  "surname": "foaf:term_surname",
+  "mbox": "foaf:term_mbox",
+  "mbox_sha1sum": "foaf:term_mbox_sha1sum",
+  "member": "foaf:term_member",
+  "homepage": "foaf:term_homepage",
+
+  "Assertion": "earl:Assertion",
+  "Assertor": "earl:Assertor",
+  "TestSubject": "earl:TestSubject",
+  "TestCriterion": "earl:TestCriterion",
+  "TestRequirement": {
+    "@id": "earl:TestRequirement",
+    "@type": "TestCriterion"
+  },
+  "TestCase": {
+    "@id": "earl:TestCase",
+    "@type": "TestCriterion"
+  },
+  "TestResult": "earl:TestResult",
+  "TestMode": "earl:TestMode",
+  "OutcomeValue": "earl:OutcomeValue",
+  "Pass": {
+    "@id": "earl:Pass",
+    "@type": "OutcomeValue"
+  },
+  "Fail": {
+    "@id": "earl:Fail",
+    "@type": "OutcomeValue"
+  },
+  "CannotTell": {
+    "@id": "earl:CannotTell",
+    "@type": "OutcomeValue"
+  },
+  "NotApplicable": {
+    "@id": "earl:NotApplicable",
+    "@type": "OutcomeValue"
+  },
+  "NotTested": {
+    "@id": "earl:NotTested",
+    "@type": "OutcomeValue"
+  },
+  "Software": "earl:Software",
+  "assertedBy": {
+    "@id": "earl:assertedBy",
+    "@type": "Assertor"
+  },
+  "subject": {
+    "@id": "earl:subject",
+    "@type": "TestSubject"
+  },
+  "test": {
```

Comment:
This isn't going to resolve correctly. Turning this:
```json
"test": "ACT-CG:rules/23a2a8"
```

Into this:

```json
"http://www.w3.org/ns/earl#test": {
  "@type": "http://www.w3.org/ns/earl#TestCriterion",
  "@value": "ACT-CG:rules/23a2a8"
}
```

You don't need to put a type here. Because of the `@vocab` that is redundant. More importantly, if you don't want a value, but to have this actually reference external data, you need to do this:

```json
"test": { "@id": "@type" }
```

This gets compacted to the following:

```json
"http://www.w3.org/ns/earl#test": {
  "@id": "https://act-rules.github.io/rules/23a2a8"
},
```

This doesn't have a type, unfortunately. In JSON-LD, you can only give `@value` a default type. If something has to resolve to an `@id`, it can't have a default type. You either have to make it part of the object, or you have to leave the type off.

Reply:
OK, I changed this but please check -- TestCriterion needs to be refined to TestCase or TestRequirement in each assertion separately.